src/components/Main/DiscordInvite/DiscordInvite.tsx

```diff
@@ -1,15 +1,15 @@
 import React from 'react';
-
-// TODO: Add text and link
-// TODO: Add graphic at the end of the page
+import {Row, Col} from 'react-bootstrap';
 
 function DiscordInvite() {
   return (
-    <div>
-      <h4>Want to join Girls Who Code at UVA?</h4>
-      <a href="#" target="_blank"><h2>Join our Discord server (arrow)</h2></a>
-      <img src="#" alt="Orange wave saying 'Get Started'" />
-    </div>
+      <Row>
+        <Col sm={12}>
+          <h4 className="text-turq pl-lg-5 ml-lg-5 pl-md-5 ml-md-5 pl-sm-5 ml-sm-5">Want to join Girls Who Code at UVA?</h4>
+          <a href="https://discord.gg/D8knkzNM8E" target="_blank"><h2 className="text-peacock discord pl-lg-5 ml-lg-5 pl-md-5 ml-md-5 pl-sm-5 ml-sm-5">Join our Discord <br />server ---{">"}</h2></a>
```

Comment:
check out these feather icons to find an arrow https://github.com/feathericons/react-feather -- arrow right looks good

check out MobileNavbar.tsx to see how you can easily integrate these icons